### PR TITLE
[567287] move rotativeImage to kitalpha core

### PR DESCRIPTION
**Pull request by tguiu**

### CODE REVIEW

File: common/features/org.polarsys.kitalpha.common.feature/feature.xml [hunk 1]
```diff
@@ -41,4 +41,11 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
```
Comment (bzendagui) [inline common/features/org.polarsys.kitalpha.common.feature/feature.xml:44]: Please update the copyright year in the header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/OSGI-INF/l10n/bundle.properties [hunk 2]
```diff
@@ -0,0 +1,18 @@
+ ###############################################################################
+ # Copyright (c) 2016 Thales Global Services S.A.S.
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/OSGI-INF/l10n/bundle.properties:2]: should this copyright year be updated? there are changes but the project name is changed !!

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/build.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+source.. = src/
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/build.properties:1]: Please add copyright header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/plugin.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/plugin.xml:1]: Please add copyright header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/schema/org.polarsys.kitalpha.common.sirius.extension.rotativeimage.exsd [hunk 5]
```diff
@@ -0,0 +1,114 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/schema/org.polarsys.kitalpha.common.sirius.extension.rotativeimage.exsd:1]: Please add copyright header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/internal/rotativeimage/Activator.java [hunk 6]
```diff
@@ -0,0 +1,83 @@
+package org.polarsys.kitalpha.common.sirius.extension.internal.rotativeimage;
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/internal/rotativeimage/Activator.java:1]: Please add copyright header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/rotativeimage/RotativeDescription.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
+package org.polarsys.kitalpha.common.sirius.extension.rotativeimage;
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/rotativeimage/RotativeDescription.java:1]: Please add copyright header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/rotativeimage/RotativeImageEditPart.java [hunk 8]
```diff
@@ -0,0 +1,342 @@
+package org.polarsys.kitalpha.common.sirius.extension.rotativeimage;
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/rotativeimage/RotativeImageEditPart.java:1]: Please add copyright header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/rotativeimage/RotativeImageEditPartProvider.java [hunk 9]
```diff
@@ -0,0 +1,29 @@
+package org.polarsys.kitalpha.common.sirius.extension.rotativeimage;
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/rotativeimage/RotativeImageEditPartProvider.java:1]: Please add copyright header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/rotativeimage/RotativeWorkspaceImageFigure.java [hunk 10]
```diff
@@ -0,0 +1,227 @@
+package org.polarsys.kitalpha.common.sirius.extension.rotativeimage;
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/src/org/polarsys/kitalpha/common/sirius/extension/rotativeimage/RotativeWorkspaceImageFigure.java:1]: Please add copyright header

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/OSGI-INF/l10n/bundle.properties (hunk 2 above)
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/OSGI-INF/l10n/bundle.properties:2]: should this copyright year be updated? there are changes but the project name is changed !!

File: common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/about.html [hunk 11]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```
Comment (bzendagui) [inline common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/about.html:1]: I think it better to use EPL 2.0 rather than 1.0

Comment (tguiu) [reply common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage/about.html:1]: License choice is done at project level. Other bundle are using EPL 1.0.

File: releng/plugins/org.polarsys.kitalpha.releng.aggregator.core/pom.xml [hunk 12]
```diff
@@ -99,6 +99,7 @@
     <module>../../../richtext/plugins/org.polarsys.kitalpha.richtext.nebula.widget</module>
     
     <!-- plugins org.polarsys.kitalpha.common.feature -->
+    <module>../../../common/plugins/org.polarsys.kitalpha.common.sirius.extension.rotativeimage</module>
```
Comment (bzendagui) [inline releng/plugins/org.polarsys.kitalpha.releng.aggregator.core/pom.xml:102]: Please update the copyright year

File: common/features/org.polarsys.kitalpha.common.feature/feature.xml (hunk 1 above)
Comment (bzendagui) [reply common/features/org.polarsys.kitalpha.common.feature/feature.xml:44]: The copyright year is not updated. please update it.

**Comment by arnauddieumegard** [comment]
This is good for me, I have functionally validated the feature

**Review by arnauddieumegard — APPROVED** [review]
Looks good to me

**Review by bzendagui — APPROVED** [review]
Ok forme.

**Pull request merged**